pymongo/encryption.py

```diff
@@ -371,7 +371,8 @@ def __init__(self, kms_providers, key_vault_namespace, key_vault_client,
 
               - `aws`: Map with "accessKeyId" and "secretAccessKey" as strings.
                 These are the AWS access key ID and AWS secret access key used
-                to generate KMS messages.
+                to generate KMS messages. An optional "sessionToken" may be
```

Comment:
Should we mention what version of `pymongocrypt` or `libmongocrypt` is needed to support this feature?

Reply:
Good idea. But I think we should just bump the version in setup.py too right? azure+gcp also are documented here and require pymongocrypt 1.1+

Reply:
I opened https://jira.mongodb.org/browse/PYTHON-2583 for this issue. I'm not going to work on it in this PR.